WISE-Community/WISE
# Fixed display name bug in discussion component

This fixes a corner-case bug where new student account names weren't showing up.

How to test
1. As Student A: register for run, start run, post to a discussion. Keep window open on the step.
2. As Student B on another browser: register for run, start run, post to the same discussion. Also reply on other students' posts.

Student A should see Student B's new posts and replies appear in real-time on their screen, with Student B's username inside the post.
